lib/qbwc/request/products.rb

```diff
@@ -165,6 +166,14 @@ def query_by_date(config, time)
 
         private
 
+        def query_inactive?(config)
+          return '' unless config['query_inactive'].to_i == 1
+
+          <<~XML
+            <ActiveStatus>asAll</ActiveStatus>
```

Comment:
Where do you see `asAll`? I see <!-- ActiveStatus may have one of the following values: ActiveOnly [DEFAULT], InactiveOnly, All -->

Reply:
what do you see ?

Reply:
lol dunno why it didn't show?
ActiveStatus may have one of the following values: ActiveOnly [DEFAULT], InactiveOnly, All